server/plugin/internal/graphql/client.go

```diff
@@ -0,0 +1,62 @@
+package graphql
+
+import (
+	"context"
+	"net/url"
+	"path"
+
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
+)
+
+type service struct {
+	client *Client
+}
+
+// Client encapsulates the third party package that communicates with Github GraphQL API
+type Client struct {
+	client   *githubv4.Client
+	org      string
+	username string
+
+	PullRequests *PullRequestService
```

Comment:
why this with capital case and rest smallcase?

Reply:
Here I am exporting only this variable that's why